src/WingetCreateCLI/Commands/UpdateLocaleCommand.cs

```diff
@@ -0,0 +1,296 @@
+
+                Manifests updatedLocales = this.PromptAndUpdateExistingLocales(originalManifests);
+                Console.WriteLine();
+                EnsureManifestVersionConsistency(originalManifests);
+                DisplayUpdatedLocales(updatedLocales);
+
+                if (string.IsNullOrEmpty(this.OutputDir))
+                {
+                    this.OutputDir = Directory.GetCurrentDirectory();
+                }
+
+                string manifestDirectoryPath = SaveManifestDirToLocalPath(originalManifests, this.OutputDir);
+
+                if (ValidateManifest(manifestDirectoryPath))
+                {
+                    if (Prompt.Confirm(Resources.ConfirmGitHubSubmitManifest_Message))
+                    {
+                        return await this.LoadGitHubClient(true) ?
+                            (commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                                originalManifests,
+                                $"Update locale: {originalManifests.VersionManifest.PackageIdentifier} version {originalManifests.VersionManifest.PackageVersion}"))
+                            : false;
+                    }
+                    else
+                    {
+                        Logger.WarnLocalized(nameof(Resources.SkippingPullRequest_Message));
+                    }
+                }
+                else
+                {
+                    return false;
+                }
+
+                return await Task.FromResult(commandEvent.IsSuccessful = true);
+            }
+            finally
+            {
+                TelemetryManager.Log.WriteEvent(commandEvent);
+            }
+        }
+
+        private static object PromptAllLocalesAsMenuSelection(DefaultLocaleManifest defaultLocale, List<LocaleManifest> localeManifests)
+        {
+            Dictionary<string, object> localeMap = new Dictionary<string, object>
+            {
+                { string.Format("{0} ({1})", defaultLocale.PackageLocale, Resources.DefaultLocale_MenuItem), defaultLocale },
+            };
+
+            foreach (var locale in localeManifests)
+            {
+                localeMap.Add(locale.PackageLocale, locale);
+            }
+
+            string selectedLocale = Prompt.Select(Resources.SelectExistingLocale_Message, localeMap.Keys.ToList());
+            return localeMap[selectedLocale];
+        }
+
+        private static void DisplayUpdatedLocales(Manifests manifest)
+        {
+            Logger.DebugLocalized(nameof(Resources.GenerateUpdatedLocalePreview_Message));
+            if (manifest.DefaultLocaleManifest != null)
+            {
+                DisplayDefaultLocaleManifest(manifest.DefaultLocaleManifest);
+            }
+
+            if (manifest.LocaleManifests != null)
+            {
+                DisplayLocaleManifests(manifest.LocaleManifests);
+            }
+        }
+
+        private Manifests PromptAndUpdateExistingLocales(Manifests originalManifests)
```

Comment:
Lets just return a tuple of the DefaultLocale and the list of locale manifests. I would also rename the parameter from `originalManifests` to `manifests` and add a method summary comment above it so it clarifies what the input and output is. My main concern is that someone will use this method thinking that the output contains the updated manifests when in fact it is only a partial manifest used solely for displaying  to console.